common/djangoapps/contentserver/test/test_contentserver.py

```diff
@@ -56,11 +62,13 @@ def setUpClass(cls):
         # A locked asset
         cls.locked_asset = cls.course_key.make_asset_key('asset', 'sample_static.txt')
         cls.url_locked = unicode(cls.locked_asset)
+        cls.url_locked_versioned = get_versioned_asset_url(unicode(cls.locked_asset))
```

Comment:
I intentionally only generate a versioned asset URL when it's unlocked because, fundamentally, we would need two configuration values to control the expiry of locked and unlocked assets.  We'd have to set a low TTL for locked assets, so that we could then set a very long TTL for the unlocked assets.

Given the number of locked assets vs unlocked assets, I made a gut decision that it's not worth it to add versioning and cache header configurability for locked files.